sift/packages/default-jre.sls

```diff
@@ -0,0 +1,2 @@
+default-jre:
```

Comment:
```suggestion
sift-default-jre:
  pkg.installed:
    - name: default-jre
```

Reply:
The reason for this is to prevent (if in the future we add support for running remnux and possible others that are built using the same process together that they won't collide.